src/app/view/components/dot-persona-selector-option/dot-persona-selector-option.component.scss

```diff
@@ -5,28 +5,40 @@
     cursor: pointer;
     display: flex;
     height: $navigation-item-height;
+    width: 100%;
 
     &:hover {
         background-color: $gray-bg;
     }
 
     dot-avatar {
-        margin-left: $basic-padding * 2;
-        margin-right: $basic-padding;
+        margin: 0 $basic-padding * 2;
     }
 
     .dot-persona-selector-option__label {
-        color: $gray-light;
+        color: $gray;
+        display: -webkit-box;
+        display: -moz-box;
         flex: 1;
         font-size: $body-font-size-base;
+        max-height: 42px;
```

Comment:
Are you sure we need this?

Reply:
this is the fallback if the ellipsis on 2nd line does not work!